.eslintrc.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   extends: [
     'eslint:recommended',
-    'plugin:prettier/recommended',
+    // 'plugin:prettier/recommended',
```

Comment:
Commenting it out temporarily because the imports sorting plugin which Felipe added was creating more issues with the bump in the version of the prettier packages.